input/docs/handbook/data-binding/xamarin-forms.md

```diff
@@ -1,7 +1,9 @@
-For Xamarin.Forms applications, you need to install the `ReactiveUI.XamForms` package and use base classes for your Views from there. Your `ContentPage` should inherit from `ReactiveContentPage<TViewModel>`, your `TextCell` should inherit from `ReactiveTextCell<TViewModel>`, etc. Those classes contain the `IViewFor<TViewModel>` interface implementation. Also, always dispose bindings via [WhenActivated](../when-activated), or else the bindings leak memory.
+For Xamarin.Forms applications you need to install the ReactiveUI.XamForms [Nuget package](https://www.nuget.org/packages/ReactiveUI.XamForms/).
```

Comment:
Maybe we should link to the README where we have a matrix of packages instead?  Only though being, a lot of issues we address have to do with people not knowing which packages to install to which projects.

Reply:
https://github.com/reactiveui/reactiveui#nuget-packages

Reply:
For Xamarin.Forms projects you only need to install the ReactiveUI.XamForms package into the Shared library right? There's no need to install it in each of the platform projects?

Reply:
Correct.  We get a lot of confusion around what package is supposed to be installed in which project.  So keeping them local to our docs for instruction helps in my mind.

Actually this is most likely where we want to link. 
https://reactiveui.net/docs/getting-started/installation/

Reply:
Are the ReactiveUI packages self-contained, or are there cases where the core libraries use bait-and-switch to tap into the platform specific code?

Reply:
Mixture of both and dynamic DLL leading of known packages.